src/renderers/shaders/ShaderChunk/lights_fragment_maps.glsl

```diff
@@ -25,11 +25,10 @@
 
 #if defined( USE_ENVMAP ) && defined( RE_IndirectSpecular )
 
-	// TODO, replace 8 with the real maxMIPLevel
```

Comment:
Just curious, this file appears to contain two copies of this `TODO` comment, but only one got removed.  Was that intentional?

Reply:
Ah, I'm just missing another one.
I'll update.

Reply:
Updated, thanks!